tests/useMeasure.test.ts

```diff
@@ -1,12 +1,13 @@
 import { renderHook, act } from '@testing-library/react-hooks';
-import useMeasure, { UseMeasureRef } from '../src/useMeasure';
+import useMeasure from '../src/useMeasure';
 
-it('by default, state defaults every value to -1', () => {
+it('by default, state defaults every value to 0', () => {
   const { result } = renderHook(() => useMeasure());
+  const [ref] = result.current;
 
   act(() => {
     const div = document.createElement('div');
-    (result.current[0] as UseMeasureRef)(div);
+    ref(div);
```

Comment:
Storing `ref` in a constant for clarity.

Reply:
Cast `ref to `UseMeasureRef` doesn't seem required any more, but happy to add it back if there's a reason for it.